internal/status/status.go

```diff
@@ -31,10 +31,10 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
 	spb "google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/protobuf/proto"
```

Comment:
This has the same problem as discussed in https://github.com/grpc/grpc-go/pull/4330

It changes the API for `WithDetails` (https://github.com/grpc/grpc-go/pull/4330#discussion_r618970752)